src/IO/Logger.cpp

```diff
@@ -75,8 +63,16 @@ void Logger::run()
 				fileStream.close();
 				lineCount = 0;
 				logId++;
-				fileStream.open(path + std::to_string(bootId) + "." + std::to_string(logId) + ext, std::ios_base::ate);
-				fileStream.sync_with_stdio(true);
+
+				// Generate the file name. Apparently it's quicker to concatenate this way than using +
+				std::string name = path;
+				name += std::to_string(bootId);
+				name += ".";
+				name += std::to_string(logId);
+				name += ext;
```

Comment:
I think this is hard to read this way, even if slightly faster.

Reply:
Yeah, good point. I'll revert it.